### DISCUSSION TITLE
Using ANDES without installation

### DISCUSSION THREAD

**Discussion by TheASingh** [CURENT/andes#346] [Q&A] [answered]
Dr. Cui,

Please suggest possible ways to work with ANDES simulation without installation?

Actually, It is not feasible for my team to go through the process of ANDES installation.

**Comment by cuihantao** [comment]
Hi,

You can search on Google about how to use a package without installing or check with your IT department.

Please post your solutions here so that they can be viewed by others.

Regards,
Hantao Cui

**Comment by TheASingh** [comment, marked as answer]
Hi all, 

To run ANDES without installing it on your team's systems, please follow below steps:

1. First, you need to have PyCharm, an IDE for python.
2. Create a new project in PyCharm. 
3. Now you need to install ANDES package. There are two ways to do:
 i. File -> Settings -> Project -> Python Interpreter. Then click on **'+'** button to add the package for ANDES.
 ii. Or you can run the **pip install ANDES** command in the terminal
5. By default, there will be a main.py file in the newly created project where you will write your code to start the simulation.
6. Now you need to install a package **pyinstaller** similar to step 3.
7. In terminal, now run the following command:
     **pyinstaller --onefile main.py**
    This command will create an exe in the dist folder of your project.
8. That's it. Now run that exe.
This exe can be shared with your team and they will be able to run it without installing ANDES.

**Reply by cuihantao** [reply]
Thank you for sharing! After reading your notes, I now understood what you mean by running without installing. It is a convenient way to share scripts that depend on ANDES.
